composer/algorithms/alibi/alibi.py

```diff
@@ -29,32 +33,33 @@ def apply_alibi(
     mask_replacement_function: Union[Callable, None],
     optimizers: Optional[Optimizers] = None,
 ) -> None:
-    """Removes position embeddings and replaces the attention function and attention mask according to `AliBi.
+    """Removes position embeddings and replaces the attention function and attention mask
+    according as per :class:`~composer.algorithms.alibi.alibi.Alibi`.
 
-    <https://arxiv.org/abs/2108.12409>`_.
+    Example: Waiting on language model test fixtures
 
     Args:
-        model: model to transform
-        heads_per_layer: number of attention heads per layer
-        max_sequence_length: maximum sequence length that the
+        model (torch.nn.Module): model to transform
```

Comment:
For the functional form of alibi, same comments regarding formatting of parameters. I think it would also be OK for each parameter that shares the same docstring to say something like `` see :class:`Alibi` `` to reduce duplication, but also cool repeating it. Since you're the first one doing algorithm docs, you should pick the convention :D